package.json

```diff
@@ -41,10 +42,20 @@
     "@commitlint/config-conventional": "^10.0.0",
     "@pact-foundation/pact": "^10.0.2",
     "@pact-foundation/pact-js-prettier-config": "^1.0.0",
+    "@tsconfig/node14": "^1.0.3",
```

Comment:
separate to this PR, but probably worth doing an update to drop 14.x and update the node engines field

Reply:
yep